packages/test/src/workflows/stack-tracer.ts

```diff
@@ -12,29 +12,29 @@ const { queryOwnWf } = wf.proxyActivities<typeof activities>({
 });
 
 export async function stackTracer(): Promise<[string, string]> {
+  const { executeChild, sleep } = wf;
```

Comment:
why is this important?

Reply:
Without this, I sometime get `Object.executeChild` and `Object.sleep` in stack traces, instead of `executeChild`/`sleep` (but not always). Not sure where the variations come from, but I don't think that we have control on this anyway.